### ISSUE TITLE
  emissary-ingress 3.7.0 not creating listeners

### ISSUE THREAD

**Issue by vyom-soft** [emissary-ingress/emissary#5129]
Hello,
I used the [helm-chart](https://github.com/emissary-ingress/emissary/tree/master/charts/emissary-ingress) to install emissary-ingress.

In the values,  [createDefaultListeners](https://github.com/emissary-ingress/emissary/blob/master/charts/emissary-ingress/values.yaml.in#L439) Is set to **true**.  

Expecting that default Listeners to be created. But unfortunately when I query for it no Listeners were found.
Can you please advise what other settings I have to adapt.
```

$ kubectl get listener --namespace emissary
No resources found in emissary namespace.
```
```
$ kubectl get deployment --namespace emissary
NAME                     READY   UP-TO-DATE   AVAILABLE   AGE
emissary-ingress         3/3     3            3           52m
emissary-ingress-agent   1/1     1            1           52m
```
```
$kubectl get deployment --namespace emissary
NAME                     READY   UP-TO-DATE   AVAILABLE   AGE
emissary-ingress         3/3     3            3           52m
emissary-ingress-agent   1/1     1            1           52m
```

Thank you.

**Comment by cindymullins-dw** [comment]
Hi @vyom-soft , can you try the Helm install from our [QuickStart](https://www.getambassador.io/docs/emissary/latest/tutorials/getting-started) and let us know?

**Comment by vyom-soft** [comment]
> Hi @vyom-soft , can you try the Helm install from our [QuickStart](https://www.getambassador.io/docs/emissary/latest/tutorials/getting-started) and let us know? The CRDs are not in Helm so you're probably missing them. The install instructions contain commands to apply the CRDs.

Hi @cindymullins-dw,

Thank you for reply.
I did read the installation instructions that CRD must be applied before the Helm chart install. And that is what the first thing I did before Helm install.
```

kubectl apply -f https://app.getambassador.io/yaml/emissary/3.7.0/emissary-crds.yaml
customresourcedefinition.apiextensions.k8s.io/authservices.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/consulresolvers.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/devportals.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/hosts.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/kubernetesendpointresolvers.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/kubernetesserviceresolvers.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/listeners.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/logservices.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/mappings.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/modules.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/ratelimitservices.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/tcpmappings.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/tlscontexts.getambassador.io created
customresourcedefinition.apiextensions.k8s.io/tracingservices.getambassador.io created
namespace/emissary-system created
serviceaccount/emissary-apiext created
clusterrole.rbac.authorization.k8s.io/emissary-apiext created
clusterrolebinding.rbac.authorization.k8s.io/emissary-apiext created
role.rbac.authorization.k8s.io/emissary-apiext created
rolebinding.rbac.authorization.k8s.io/emissary-apiext created
service/emissary-apiext created
deployment.apps/emissary-apiext created
```

_And I did apply CRD again._

```
$ kubectl apply -f https://app.getambassador.io/yaml/emissary/3.7.0/emissary-crds.yaml
customresourcedefinition.apiextensions.k8s.io/authservices.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/consulresolvers.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/devportals.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/hosts.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/kubernetesendpointresolvers.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/kubernetesserviceresolvers.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/listeners.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/logservices.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/mappings.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/modules.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/ratelimitservices.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/tcpmappings.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/tlscontexts.getambassador.io configured
customresourcedefinition.apiextensions.k8s.io/tracingservices.getambassador.io configured
namespace/emissary-system unchanged
serviceaccount/emissary-apiext unchanged
clusterrole.rbac.authorization.k8s.io/emissary-apiext unchanged
clusterrolebinding.rbac.authorization.k8s.io/emissary-apiext unchanged
role.rbac.authorization.k8s.io/emissary-apiext unchanged
rolebinding.rbac.authorization.k8s.io/emissary-apiext unchanged
service/emissary-apiext unchanged
deployment.apps/emissary-apiext unchanged
```

**Comment by cindymullins-dw** [comment]
Ok, I see.  I found a [previous similar issue](https://github.com/emissary-ingress/emissary/issues/3894) and the problem there was they were setting the env var AMBASSADOR_LABEL_SELECTOR.  It sounds like that was blocking Envoy from being configured properly.  Is that a possible cause?

**Comment by cindymullins-dw** [comment]
Are you still having issues with this?  We do have a weekly live help session on Thursdays for Emissary at 2:30pm ET.  Let me know if you'd like to join sometime.

**Comment by cindymullins-dw** [comment]
Wonder if the Listeners could be in the default namespace?  Could you try running `kubectl get listener -A`?

**Comment by vyom-soft** [comment]
Thank you.

I followed  [emissary-ingress-quick-start](https://www.getambassador.io/docs/emissary/latest/tutorials/getting-started#emissary-ingress-quick-start) 

I deployed the emissary CRD v3.9.1 in a namespace called emissary 
Listeners deployed in  namespace emissary. via kubectl apply -f (Not via Helm, createDefaultListeners: false)
And latest emissary-ingress also in the same namespace emissary.
so it works now.
